docs/Using Fleet/enroll-hosts.md

```diff
@@ -1,316 +1,111 @@
-    - [Configure and launch osquery](#configure-and-launch-osquery)
-      - [Launching osqueryd using command-line flags](#launching-osqueryd-using-command-line-flags)
-      - [Launching osqueryd using a flag file](#launching-osqueryd-using-a-flag-file)
-    - [Migrating from plain osquery to osquery installer](#migrating-from-plain-osquery-to-osquery-installer)
-      - [Generate installer](#generate-installer)
-      - [Migrate](#migrate)
-  - [Add Chromebooks with the fleetd Chrome extension](#add-chromebooks-with-the-fleetd-chrome-extension)
-  - [Grant full disk access to osquery on macOS](#grant-full-disk-access-to-osquery-on-macos)
-    - [Creating the configuration profile](#creating-the-configuration-profile)
-      - [Obtaining identifiers](#obtaining-identifiers)
-      - [Creating the profile](#creating-the-profile)
-      - [Test the profile](#test-the-profile)
+- [How to enroll hosts?](#how-to-enroll-hosts)
+  - [CLI](#cli)
+  - [UI](#ui)
+    - [Generate installer to enroll host to a specific team](#generate-installer-to-enroll-host-to-a-specific-team)
+  - [Enroll multiple hosts](#enroll-multiple-hosts)
+  - [Including Fleet Desktop](#including-fleet-desktop)
+- [Enroll Chromebooks with the fleetd Chrome extension](#add-chromebooks-with-the-fleetd-chrome-extension)
+  - [Overview](#overview-2)
+  - [Step 1: OU for Chromebook users](step-1-ou-for-chromebook-users)
+  - [Step 2: OU to block non-Chromebook devices](step-2-ou-to-block-non-chromebook-devices)
+- [Grant full disk access to osquery on macOS](#grant-full-disk-access-to-osquery-on-macos)
+  - [Creating the configuration profile](#creating-the-configuration-profile)
+    - [Obtaining identifiers](#obtaining-identifiers)
+    - [Creating the profile](#creating-the-profile)
+    - [Test the profile](#test-the-profile)
+- [Advanced](#advanced)
+  - [Signing installers](#signing-installers)
+  - [Generating Windows installers using local WiX toolset](#generating-windows-installers-using-local-wix-toolset)
+  - [fleetd configuration options](#configuration-options)
+  - [Enroll hosts with plain osquery](#enroll-hosts-with-plain-osquery)
 
 ## Introduction
 
-Fleet gathers information from an [osquery](https://github.com/osquery/osquery) agent installed on each of your hosts. The recommended way to install osquery is using Fleetd, which includes [Orbit](https://fleetdm.com/docs/using-fleet/fleetd), Fleet's lightweight osquery runtime and auto-updater.
+Fleet gathers information from an [osquery](https://github.com/osquery/osquery) agent installed on each of your hosts. The recommended way to install osquery is using fleetd.
 
-You can also install plain osquery on your hosts and connect to Fleet using osquery's `TLS API` plugins.
+You can also [install plain osquery](https://fleetdm.com/docs/using-fleet/enroll-hosts#enroll-hosts-with-plain-osquery) on your hosts and connect to Fleet using osquery's `TLS API` plugins.
```

Comment:
```suggestion
```

Should we remove this sentence because the instructions for plain osquery are now in the contributor docs?

Reply:
We should remove this, I added a section on the bottom of the doc that will link to contributor docs, so if someone is searching for plain osquery can find a link to contributor docs.